PSKoans/Koans/Foundations/AboutArrays.Koans.ps1

```diff
@@ -109,4 +185,78 @@ Describe 'Arrays' {
 
         # NOTE: The above will actually throw errors if you have PowerShell running in Strict Mode.
     }
+
+    It 'creates arrays from a range' {
+        # The .. notation used to reverse an array may be used to array.
+
+        $Array = 1, 2, 3, 4, 5
+
+        $StartIndex = __
+        $StartIndex..5 | Should -Be $Array
+    }
+
+    It 'an array of letters may be created from a range' {
+        # An array of characters, or letters, can be created.
+
+        $firstLetter = '__'
+        $lastLetter = '__'
+
+        if ($PSVersionTable.PSEdition -eq 'Core') {
+            # PowerShell Core uses .. between the letters to create an array.
+
+            $letters = $firstLetter..$lastLetter
+
+            $letters | Should -Be 'a', 'b', 'c', 'd'
+        }
+        else {
+            # Windows PowerShell 5 and below have to work a lot harder to achieve the same thing.
+
+            $letters = [Int][Char]$firstLetter..[Int][Char]$lastLetter -as [Char[]]
```

Comment:
```suggestion
            $letters = ([Int][Char]$firstLetter)..([Int][Char]$lastLetter) -as [Char[]]
```
Even if PS understands this implicitly (which tbh I kinda doubt), I doubt newbies will 😂